openzwave-sys/openzwave-c/value_classes/value_id.h

```diff
@@ -8,29 +8,38 @@ typedef OpenZWave::ValueID ValueID;
 typedef ValueID::ValueGenre ValueGenre;
 typedef ValueID::ValueType ValueType;
 
+typedef struct cValueID {
```

Comment:
ok I think I understood; I'll look at it once again with a clearer mind after this night.